client/styles/bookingButton.css

```diff
@@ -0,0 +1,19 @@
+.button {
+  background-color: #FF5A5F;
+  border: none;
+  border-radius: 4px;
+  color: white;
+  cursor: pointer;
+  font-size: 16px;
+  padding-top: 12px;
```

Comment:
Consider using shorthand syntax if top and bottom have same padding

Reply:
Good catch!